### PR TITLE
Add italian language support

### PR DESCRIPTION
**Pull request by iidoki**
italian translation

### CODE REVIEW

**Comment by TukanDev** [comment]
No way laura did it woooo

File: src-tauri/lang/it.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    "gc_dev_data": "Scarica i dati della build in sviluppo di Grasscutter più recente, che non include il file jar. È utile per aggiornare.",
+    "encryption": "Di solito questo dovrebbe essere disabilitato.",
+    "resources": "Queste sono anche necessari per eseguire un server Grasscutter. Questo bottone è grigio se hai una cartella delle risorse esistente con contenuto all'interno.",
+    "emergency_rsa": "In caso qualcosa è andato storto, forza l'eliminazione dell'RSA.",
+    "use_proxy": "Usa il proxy interno di Cultivation. Dovresti averlo attivato a meno che tu non usi qualcosa come Fiddler.",
+    "patch_rsa": "Patch e unpatch l'RSA del gioco automaticamente. A meno che tu non usi versioni vecchie/non ufficiali del gioco (3.0 e precedenti), questo dovrebbe essere attivato.",
+    "add_delay": "Imposta il ritardo nel caricatore 3dmigoto! \nQuesto dovrebbe risolvere i problemi di caricamento, ma aggiungerà un piccolo ritardo al caricamento di 3dmigoto all'avvio del gioco. \nOra puoi avviare il gioco con 3dmigoto di nuovo.",
+    "migoto": "For importing models from GameBanana"
```
Comment (SpikeHD) [inline src-tauri/lang/it.json:87]: Was this one missed?

Comment (iidoki) [reply src-tauri/lang/it.json:87]: whoopsies, yup

Comment (iidoki) [reply src-tauri/lang/it.json:87]: i hate myself

**Comment by iidoki** [comment]
updated it now, lmk if i missed something else 👍

**Pull request merged**